### DISCUSSION TITLE
Account Geth x MiningCore Question?

### DISCUSSION THREAD

**Discussion by modanezi** [coinfoundry/miningcore#1272] [Q&A]
Good morning, I'm trying to set up a pool and I have some questions about accounts and payment methods.

In Geth when I do the command

geth account new

He creates the account and it stays on geth,

> eth.coinbase
"0xCf5767AC46EA2A9162F0734651d6cF17e5BfcF10"
>

**What is this account used for? Does she receive the mining and make the payments?**

in MiningCore I have these settings

    "pools": [
            "address": "0xPOOL_ADDRESS",
            "rewardRecipients": [
                {
                    "type": "op",
                    "address": "0xPOOL_ADDRESS",

            "paymentProcessing": {
                "enabled": false,
                "minimumPayment": 0.01,
                "minimumPaymentToPaymentId": 5,
                "payoutScheme": "PPLNS",
                "payoutSchemeConfig": {
                    "factor": 2
                },
                "coinbasePassword": "<your wallet password if you are running parity with wallet locked>",
                "keepUncles": false,
                "keepTransactionFees": false

**My question is, who makes the payment on the blockchain (geth or miningcore)? who sends the coins to the miners? How does unlocking work?**

Thanks a lot for the help!!

**Comment by ei8ht187** [comment]
The option "coinbasePassword" does not work for geth. Therefore unlock your account just use geth with the --password statement

**Reply by oliverw** [reply]
Exactly.
